monitor.py

```diff
@@ -775,10 +778,19 @@ class ReaperMonitor(Monitor):
     * MAX_AGE - A datetime.timedelta or number of days representing
     the time that must pass before an item can be safely deleted.
 
+    A subclass of ReaperMonitor MAY define values for the following constants:
+    * BATCH_SIZE - The number of rows to fetch for deletion in a single
+      batch. The default is 1000.
+
+    If your model class has fields that might contain a lot of data
+    and aren't important to the reaping process, put their field names
+    into a list called LARGE_FIELDS and the Reaper will avoid fetching
+    that information, improving performance.
     """
     MODEL_CLASS = None
     TIMESTAMP_FIELD = None
     MAX_AGE = None
+    BATCH_SIZE = 1000
```

Comment:
The batch size should make the memory problem Robert was seeing go away in general, without us having to fine-tune every single reaper.